src/Tools/AI Test Toolkit/src/Logs/AITLogEntries.Page.al

```diff
@@ -68,6 +68,23 @@ page 149033 "AIT Log Entries"
                 {
                     StyleExpr = StatusStyleExpr;
                 }
+                field(Accuracy; Rec.Accuracy)
+                {
+                }
+                field("No. of Turns Passed"; Rec."No. of Turns Passed")
+                {
+                    Visible = false;
+                }
+                field("No. of Turns Executed"; Rec."No. of Turns Executed")
```

Comment:
Executed is a bad term. See to update it, there is a similar comment in a different place.